### PR TITLE
 [Mission] Affichage des actions Env dans la timeline du formulaire de mission

### LINKED ISSUE #3023: Synchro missions – Afficher les actions env dans la timeline du formulaire de mission
Pour cette V1, on peut n'afficher que le type d'action (contrôle, surveillance, note) et ne pas récupérer le détail des actions. 
Les cartes d'actions ne sont pas cliquables. 
Elles se différencient des actions pêche
- par le fait qu'elles n'ont pas de bouton "supprimer" et "dupliquer"
- par une petite mention "action CACEM" juste en-dessous de la carte  

[Ecran XD ici](https://xd.adobe.com/view/bcb50441-8561-431e-9ce7-1a78965b60e4-e9e6/screen/6b079992-b75a-40a9-af17-0dcd626cd640) (les détails de l'action dans la carte ne sont pas affichés)

### PR DESCRIPTION
**Pull request by louptheron**
## Linked issues

- Resolve #3023 

----

- [x] Tests E2E (Cypress)

## Summary by CodeRabbit

- **Refactor**
	- Updated ESLint configuration for improved code quality checks.
	- Standardized import statements for `Mission` across test files for consistency.
	- Comprehensive update across numerous files to consolidate and update import paths, reflecting a reorganization of file structure for better maintainability.
- **Tests**
	- Added a new test case for verifying the display of environmental actions in the Side Window Mission Form Action List.
- **Style**
	- Adjusted styling properties in components for enhanced user interface consistency.
- **New Features**
	- Introduced a new `EnvMissionAction` type for better handling of environmental actions in missions.
- **New Features**
	- Added a new React component `ActionCard` to display mission action details with interactive selection functionality.
- **New Features**
	- Introduced a new component `EnvActionCard` to display environmental mission actions with corresponding icons and labels.
- **New Features**
	- Refactored the `FishActionCard` component to focus on displaying action details and providing options for duplicating and removing actions.
- **New Features**
	- Updated the `ActionList` component with new imports, hooks, props, and modified logic for sorting and displaying mission actions.
- **New Features**
	- Added styled components for `ActionLabel` and `Head` in the `ActionList` component for improved UI design.

### CODE REVIEW

File: frontend/src/features/Mission/components/MissionForm/MainForm/constants.tsx [hunk 1]
```diff
@@ -1,14 +1,4 @@
+import { Mission } from '@features/Mission/mission.types'
 import { getOptionsFromLabelledEnum } from '@utils/getOptionsFromLabelledEnum'
-import { Mission } from 'domain/entities/mission/types'
 
 export const MISSION_TYPES_AS_OPTIONS = getOptionsFromLabelledEnum(Mission.MissionTypeLabel)
-
-export const MISSION_EVENT_UNSYNCHRONIZED_PROPERTIES_IN_FORM = [
```
Comment (claire2212) [inline frontend/src/features/Mission/components/MissionForm/MainForm/constants.tsx:6]: pourquoi cette suppression ?

File: frontend/src/features/Mission/components/MissionForm/ActionList/MonitorEnvMissionActionItem.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  align-items: center;
+  color: ${p => p.theme.color.slateGray};
+  display: flex;
+  font-size: 13px;
+  /* This padding allows the top 2px outline to be visible in InnerWrapper */
+  padding-top: 2px;
+  user-select: none;
+`
+
+const DateLabel = styled.div`
```
Comment (claire2212) [inline frontend/src/features/Mission/components/MissionForm/ActionList/MonitorEnvMissionActionItem.tsx:93]: y'a pas moyen de mutualiser le style entre ce fichier et `MissionActionItem.tsx` ? j'ai l'impression qu'il y a des doublons

**Review by claire2212 — APPROVED** [review]

File: frontend/src/features/Mission/components/MissionForm/MainForm/constants.tsx (hunk 1 above)
Comment (louptheron) [reply frontend/src/features/Mission/components/MissionForm/MainForm/constants.tsx:6]: C'était un doublon, cette propriété existe ici : `'@features/Mission/components/MissionForm/sse`

File: frontend/src/features/Mission/components/MissionForm/ActionList/index.tsx [hunk 3]
```diff
@@ -23,12 +29,46 @@ type ActionListProps = Readonly<{
...
+      ...action,
+      index,
+      source: Mission.MissionSource.MONITORFISH
+    }))
+
+    if (!getMissionApiQuery.data) {
+      return monitorFishActions
+    }
+
+    const monitorEnvActions = getMissionApiQuery.data.envActions.map(action => ({
```
[the remark this answers is not in the document]
Comment (louptheron) [reply frontend/src/features/Mission/components/MissionForm/ActionList/index.tsx:54]: Sur les hooks RTK il n'y a pas besoin, et dans ce cas s'il y a un catch ça sera géré dans le `initiate` qui affiche le formulaire (car cette query récupère juste le cache)

**Pull request merged**